### DISCUSSION TITLE
Can we extend @collections classes ?

### DISCUSSION THREAD

**Discussion by dgandhi17** [isar/isar#50] [Help]
@leisim

**Comment by simc** [comment]
@dgandhi17 I thought about it but it's not clear what the goal should be. Maybe you could explain your use case. Should multiple different types be stored in a single collection (for example `Cat` and `Dog` in `Pet` collection) or do you want to store a single type in different collections (for example `Pet` in `Cat` collection and in `Dog` collection)?

**Reply by dgandhi17** [reply]
Well I have label collection that I want extend in addressLabel table and fieldLabel collection.

You mean i can store another collection in one collection as column ?
